src/inference/include/openvino/runtime/properties.hpp

```diff
@@ -646,15 +646,27 @@ static constexpr Priorities priorities{"MULTI_DEVICE_PRIORITIES"};
  * @brief Type for property to pass set of properties to specified device
  * @ingroup ov_runtime_cpp_prop_api
  */
-struct Properties {
+
+struct Properties : public Property<std::map<std::string, std::map<std::string, Any>>> {
+    using Property<std::map<std::string, std::map<std::string, Any>>>::Property;
+
+    /**
+     * @brief Constructs property
+     * @param configs set of property values with names
+     * @return Pair of string key representation and type erased property value.
+     */
+    inline std::pair<std::string, Any> operator()(const AnyMap& config) const {
+        return {name(), config};
+    }
```

Comment:
just for understanding: for what cases it can be useful?
I supposed originally `ov::device::properties` is designed to pass secondary properties for a single device (here I mean setting params to compile_model, query_model and similar).

is it needed for cases like _extract properties for HETERO / AUTO subgraphs / models using a single call_?

Reply:
@ilya-lavrenov In general yes. If user want to set properties by one line he can use API which is already available. But in some cases it can be easy to use new one. For example, in benchmark_app where logic for settings is complicated, I'm using new one API. So if user already has map per device, he can just use it.